### DISCUSSION TITLE
Is there any data collected by the swift-identified-collections library?

### DISCUSSION THREAD

**Discussion by stremsdoerfer** [pointfreeco/swift-identified-collections#64] [Q&A] [answered]
Hi !

In order for us to provide accurate Privacy Nutrition Labels to Apple during our review, we need to know whether all our third party libraries collect any kind of data. Is there any data collected by the swift-identified-collections library that we should specify in our review? (such as crash, usage, diagnostics etc.)

Thanks!

**Comment by mbrandonw** [comment, marked as answer]
Hi @stremsdoerfer, no, there is no communication with the outside world in this library. But also you probably shouldn't take our word for it. You should verify it by looking at the library's source code.

**Reply by stremsdoerfer** [reply]
Thank you!
